.github/workflows/mingw-w64.yml

```diff
@@ -0,0 +1,65 @@
+name: mingw-w64
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        with-patch:
+          - true
+          - false
+    env:
+      MINGW_PACKAGE_PREFIX: mingw-w64-ucrt-x86_64
+      MSYSTEM: UCRT64
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: dependencies
+        shell: c:\msys64\usr\bin\bash.exe -e -l {0}
+        run: |
+          pacman \
+            --needed \
+            --noconfirm \
+            --sync \
+            ${MINGW_PACKAGE_PREFIX}-cc \
+            ${MINGW_PACKAGE_PREFIX}-cmake \
+            ${MINGW_PACKAGE_PREFIX}-curl \
+            ${MINGW_PACKAGE_PREFIX}-ninja \
+            ${MINGW_PACKAGE_PREFIX}-openssl \
+            ${MINGW_PACKAGE_PREFIX}-zlib
+      - name: patch
```

Comment:
This step applies only https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-aws-sdk-cpp/aws-sdk-cpp-pr-1333.patch .
https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-aws-sdk-cpp/0001-fix-dll-import-libary-folder.patch isn't applied.

Because we need only the former for passing build. The latter is for fixing install location for shared library.

For now, the former patch is for static build. We can't build aws-sdk-cpp for shared library with the former patch. So we don't need the latter patch yet.